datasets/br/br_tcu_debarred/br_tcu_debarred.yaml

```diff
@@ -0,0 +1,21 @@
+title: Brazil's TCU list of Debarred Bidders
```

Comment:
just a reminder here too about the directory structure: br/tcu_debarred/br_tcu_debarred